core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java

```diff
@@ -237,6 +237,9 @@ private TestingTrinoServer(
             // TODO: enable failure detector
             serverProperties.put("failure-detector.enabled", "false");
             serverProperties.put("catalog.store", "none");
+
+            // Reduce memory footprint in tests
+            serverProperties.put("query.min-expire-age", "5s");
```

Comment:
aren't there tests which depend on this being longer. Getting query stats or sth?

Reply:
Good question. Not sure how can I know that.
Even if i set this to 0, I won't deterministically find these tests, right?

Reply:
If test pass - the I guess we are good :)